api-server/README.md

```diff
@@ -2,30 +2,21 @@
 
 Backend API powering [Infinite Industries](https://infinite.industries).
 
-## Features
-
-- **TODO**: [refer to Project Page] (https://github.com/infinite-industries/infinite/projects/1)
-
-## Development Environment Setup
-
-### Dependencies
-
-You will need the following tools:
-
-- [Node.js -- Install with NVM](https://github.com/nvm-sh/nvm): 16.x.x or higher
-- [npm](https://www.npmjs.com/get-npm): 8.x.x or higher
-- [Postgres](https://www.postgresql.org/): 9.4.0 or higher
-- [docker](https://www.docker.com)
-
-### Setup
-
-Set up the env file and add *1nfinite.pem* file to keys directory. Sample files are provided with a `.sample` suffix.
-A core team member can send you the dev secrets via the Infinite Industries Slack.
-
- ```bash
- cp .env.sample .env
- cp keys/1nfinite.pem 1nfinite.pem    
- ```
+* [Installation](#installation)
```

Comment:
I like the table of contents approach here

Reply:
Here's the shell script - no extra dependencies needed - https://github.com/ekalinin/github-markdown-toc/blob/master/gh-md-toc